pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -146,9 +147,13 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tp *v1alpha1.TektonPipel
 			if err != nil {
 				return err
 			}
+
+			// TODO: visit the metrics condition for upgrade
+			// whether to use operator version or component version
```

Comment:
Not sure I understand this one

Reply:
we use component version for metrics but I wasn't sure how that is used
so added to TODO to visit that again
if we can operator version in that ?
because if component version is same in a new release then upgrade metrics will not logged ! as per current logic
cc @savitaashture